src/main/java/com/smartbear/ready/plugin/jira/factories/JiraPrefsFactory.java

```diff
@@ -97,7 +98,7 @@ public SimpleForm getForm() {
         if (form == null) {
             form = new SimpleForm();
             form.addSpace();
-            JTextField loginField = form.appendTextField(BUG_TRACKER_LOGIN, BUG_TRACKER_LOGIN_DESCRIPTION);
+            JTextField loginField = form.appendTextField(BUG_TRACKER_LOGIN_CAPTION, BUG_TRACKER_LOGIN_DESCRIPTION);
```

Comment:
I believe setFormValues() method needs text field name to match the key in StringToStringMap returned by getValues(). But getValues() puts login value under BUG_TRACKER_LOGIN key.

Reply:
thanks for good catch, fixed